frontend/src/components/Stores.vue

```diff
@@ -0,0 +1,197 @@
+<template>
+  <div class="container">
+    <div style="min-width:1115px" class="card mt-5">
+      <h2 class="card-header text-center">Store Information</h2>
+      <div style="text-align: center" class="m-3" v-cloak>
+        <p v-if="stores.length > 0">showing {{ stores.length }} items</p>
+        <p v-if="!dataIsAvailable">Server side data is not ready yet. Should auto refresh soon!</p>
+        <a v-if="dataIsAvailable" href="/api/acme.csv">Download information as CSV</a>
+      </div>
+      <div style="padding-bottom: 20px" v-if="dataIsAvailable && stores" class="d-flex justify-content-center">
+        <v-pagination
+            v-model="page"
+            :pages="numberOfPages"
+            :range-size="0"
+            active-color="#DCEDFF"
+            @update:modelValue="updatePageHandler"
+        />
+      </div>
+      <table v-if="dataIsAvailable" class="table table-striped">
+        <thead class="thead-light text-center">
+        <tr>
+          <th>Store Id</th>
+          <th>Store Code</th>
+          <th>Store Description</th>
+          <th>
+            <div class="row d-flex justify-content-center" style="max-width: 250px">
+              <span>Store Name</span>
+              <input class="col-6" v-model="nameFilter" placeholder="name filter">
+              <div class="col-3">
+                <button class="btn btn-sm btn-outline-dark" @click="fetchStoresPage(page)">Apply</button>
+              </div>
+              <div class="col-2">
+                <button style="padding: 10px" class="btn btn-close btn-sm" @click="resetFilter"></button>
+              </div>
+            </div>
+          </th>
+          <th>Store Opening Date</th>
+          <th>Store Type</th>
+          <th>Seasons</th>
+          <th>Additional Information</th>
+        </tr>
+        </thead>
+        <tbody v-cloak>
+        <tr v-for="s in stores" :key="s.id">
+          <td class="col-1">
+            <info-or-null :condition="s.id">{{ s.id }}</info-or-null>
+          </td>
+          <td class="col-1" style="max-width: 50px">
+            <info-or-null :condition="s.code">{{ s.code }}</info-or-null>
+          </td>
+          <td class="col-1">
+            <info-or-null :condition="s.description">
+              <a class="" v-if="!s.readMoreActivated" @click="activateReadMore(s)" href="#">
```

Comment:
semantically is not a good practice to have on click events on `a` elements. this should be a button